From 720dc5d264b027fd47911bbc00f5af0e162de70c Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Mon, 09 Jan 2023 15:15:48 +0200
Subject: [PATCH] [SW-0] gaudi2: used tabs for indentation

Use tabs for indentation instead of spaces and a mix of two.

Change-Id: Ic28cfb33f795626c983b2fc22bc1c83a141b2c9b
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/arch/gaudi2/common/hl.h b/arch/gaudi2/common/hl.h
index d3c74a6..7c745ea 100644
--- a/arch/gaudi2/common/hl.h
+++ b/arch/gaudi2/common/hl.h
@@ -11,10 +11,10 @@
 
 #define BOOT_DESC_MAX_IMGS	5
 
-#define HL_PLL_DIV_FACTOR       0x200
-#define HL_PLL_DIV_FACTOR_CMD   0x220
-#define HL_PLL_DIV_SEL	        0x280
-#define HL_PLL_DIV_EN	        0x2a0
+#define HL_PLL_DIV_FACTOR	0x200
+#define HL_PLL_DIV_FACTOR_CMD	0x220
+#define HL_PLL_DIV_SEL		0x280
+#define HL_PLL_DIV_EN		0x2a0
 
 void set_pci_msix_pba_offset_val(void);
 void set_slv_cfg_reg_val(void);
diff --git a/arch/gaudi3/arc/onix.c b/arch/gaudi3/arc/onix.c
index d38b4ff..9c8776e 100644
--- a/arch/gaudi3/arc/onix.c
+++ b/arch/gaudi3/arc/onix.c
@@ -45,7 +45,7 @@
  * 0x0201_0002_0000_0000,
  * change SRAM extension for ARC1 accordingly.
  */
-#define SRAM_EXTENSION_MSB      0x2010002
+#define SRAM_EXTENSION_MSB	0x2010002
 
 void sram_set_extension(void)
 {
